### ISSUE TITLE
Missing Talkback descrptions for back buttons in conversations

### ISSUE THREAD

**Issue by clado** [signalapp/Signal-Android#12546]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Two back buttons related to the conversation view are missing Talkback descriptions

### Steps to reproduce

* turn Talkback on (in System Settings)
* navigate to conversation view

**Actual result:** back button is unlabeled
**Expected result:** back button should be labeled

* navigate to conversation settings

**Actual result:** back button is unlabeled
**Expected result:** back button should be labeled

### Screenshots

first button, in conversation view:
![a screenshot of the signal conversation view, with some sensitive information visibly censored. the back button in the upper left-hand corner has a green Talkback focus](https://user-images.githubusercontent.com/2933589/197375430-5e03c232-1613-415e-89a1-884e3b08c348.png)

second button, in conversation settings view:
![a screenshot of the signal conversation settings view, with some sensitive information visibly censorsed. the back button in the upper left-hand corner has a green Talkback focus](https://user-images.githubusercontent.com/2933589/197375431-25f48a27-3296-46b4-8e4d-a7bef83a114f.png)

### Device info
**Device:** Google Pixel 2
**Android version:** Android 12 (LineageOS 19)
**Signal version:** 5.53.5

### Link to debug log
don't want to do this 😛 <3
